src/server/apis/dash-data.ts

```diff
@@ -307,14 +307,18 @@ export async function fetchIncomingData(
       }
 
       if (newCommits.length && relevantReview) {
+        // Commit can be null.
+        const url = relevantReview.commit ?
```

Comment:
would be make more sense to set the URL to null here and have the UI not create an anchor tag (I'm assuming that's what it's doing).

Reply:
It doesn't create an anchor tag.